conference/react-js/src/Routers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Upcomming from './components/Upcomming';
+import Past from './components/Past';
+
+const Routers = () => (
+  <Router>
+    <div className="Container home">
+      <ul className="tabs-list">
```

Comment:
Move this list to the separate component (Navigation)

Reply:
maybe it's not navigation , it's tabs or something else?